nexus/types/src/deployment.rs

```diff
@@ -840,38 +807,6 @@ impl fmt::Display for BlueprintZoneDisposition {
     }
 }
 
-/// Filters that apply to blueprint zones.
```

Comment:
I think the intention behind this was good at the time, but our changes have shown it to not be strictly necessary. Repeating myself, I'm happy to be rid of the extra code and maintenance burden here, however small it may have been.